client/api/course.js

```diff
@@ -39,11 +40,16 @@ function publishRepositoryMeta(id) {
   return request.post(urls.publish(id)).then(res => res.data);
 }
 
+function addTag(courseId, data) {
```

Comment:
Don't forget the removal ;) Merge the latest develop and move from notion of course (repositoryId). This file is now named `api/repository`.